bats/tests/pyrsia_cli.bats

```diff
@@ -79,6 +79,44 @@ setup() {
   assert_output --partial '7888'
 }
 
+@test "Testing 'pyrsia config edit' CLI, check if the config can be changed." {
+  # change hostname
+  run "$PYRSIA_CLI" config edit --host host.for.test
+  assert_output --partial 'Node configuration Saved'
+
+  # revert the change
+  run "$PYRSIA_CLI" -c -e --host localhost
+  refute_output --partial 'Invalid'
+
+  # change port
+  run "$PYRSIA_CLI" config edit --port 9999
+  assert_output --partial 'Node configuration Saved'
+  # revert the change
+  run "$PYRSIA_CLI" -c -e --port 7888
+  refute_output --partial 'Invalid'
+
+  # change diskspace
+  run "$PYRSIA_CLI" config edit --diskspace "5 GB"
+  assert_output --partial 'Node configuration Saved'
+  # revert the change
+  run "$PYRSIA_CLI" -c -e --diskspace "10 GB"
+  refute_output --partial 'Invalid'
+
+  # change two or more values at once
+  run "$PYRSIA_CLI" config edit --host 192.168.0.0 --port 9999 --diskspace "5 GB"
+  assert_output --partial 'Node configuration Saved'
```

Comment:
A better test case may be after updating using `pyrsia config edit`, use `pyrsia config --show` to get the values and compare in case if it is achievable.

Reply:
@dasmanas Thank you. Yes... I added some codes to confirm the settings.